.github/workflows/test-docker.yml

```diff
@@ -0,0 +1,96 @@
+name: Test in docker
+
+on:
+  push:
+
+jobs:
+  test:
+    runs-on: ['ubuntu-22.04']
+
+    strategy:
+      fail-fast: false
+      matrix:
+        tarantool:
+          - '1.10'
+          - '2.10'
+          - '2.11'
+
+        image:
+          - ubuntu:focal
+          - ubuntu:jammy
+          - debian:buster
+          - debian:bullseye
+
+        # Only 2.11 is available on debian bookworm
+        include:
+          - tarantool: '2.11'
+            image: debian:bookworm
+
+    container:
+      image: ${{ matrix.image }}
+
+    env:
+      TARANTOOL_CACHE_KEY_SUFFIX: -DA-${{ github.run_id }}
+      DEBIAN_FRONTEND: noninteractive
+      TZ: Etc/UTC
+
+    steps:
+      - name: Setup system
+        run: |
+          apt-get -y update
+          apt-get -y upgrade
+          apt-get -y install --no-install-recommends --no-install-suggests \
+            tzdata sudo lsb-release gnupg ca-certificates rsync
+
+      - name: Setup nodejs
+        uses: actions/setup-node@v3
+        with:
+          node-version: '18.18.0'
+
+      - uses: actions/checkout@v4
+
+      - id: get-latest
+        run: |
+          node <<'SCRIPT'
+            process.env["INPUT_TARANTOOL-VERSION"] = "${{ matrix.tarantool }}"
+            require("./dist/main").latest_version("${{ matrix.tarantool }}").then(v => {
+              console.log(v)
+              require("./dist/main").setActionOutput('version', v);
+            })
+          SCRIPT
```

Comment:
Nit: It can be replaced with the `latest-version` action to reduce code duplication and to make the workflow code concise.

https://github.com/tarantool/setup-tarantool/blob/615e8f4e4dfbf4e41631a72660d27f1ede102aa0/.github/workflows/test.yml#L123-L126

Reply:
Now I understood that we need to [adjust](https://github.com/tarantool/setup-tarantool/pull/45#discussion_r1501697662) the action for determining the latest 2.10 release (not just latest series-2 release). I'm OK if you just leave it as is (as well as all the other comments that are marked as nits). Just let me know about your decision.

Reply:
for some reason the packages versioning is different for 1.10 and 2.10 (2.11) releases:
1.10
package: 1.10.15.0.ge14d0b4ea-1
abc: 1.10.15
abcd: 1.10.15.0
abc-d: 1.10.15-0
git-describe: 1.10.15-0-ge14d0b4ea

2.10 and 2.11
package: 2.10.8.g4472d9247f-1
abc: 2.10.8
abcd: 2.10.8.g4472d9247f
abc-d: 2.10.8-g4472d9247f
git-describe: 2.10.8-g4472d9247f-1

Reply:
The reason is the [new release policy](https://github.com/orgs/tarantool/discussions/6182#user-content-deb-package-versions-debian-ubuntu) that is in effect since 2.10.0. Now I remember that I left 2.10+ support in the `latest-version` action unimplemented, because it was not needed for my test cases.

https://github.com/tarantool/setup-tarantool/blob/615e8f4e4dfbf4e41631a72660d27f1ede102aa0/.github/actions/latest-version/action.yml#L28-L32

Reply:
I've fixed it in the action itself:
https://github.com/tarantool/setup-tarantool/pull/45/commits/78abdce492f16a4ef0c6411840bc6c39124a8c9b

Reply:
Nice, thank you!